run_analysis.R

```diff
@@ -53,4 +56,4 @@ dataset.tidy <- dcast(dataset.melt, activitylabel + subject ~ variable, mean)
 
 
 #creating a tidy dataset file  
-write.table(dataset.tidy, file = "tidydataset.txt" row.names = FALSE)
+write.table(dataset.tidy, file = "tidydataset.txt", row.names = FALSE)
```

Comment:
Simple missing comma here; in combination with the change above I was able to run your script.